src/components/AccountList.tsx

```diff
@@ -65,8 +63,6 @@ const AccountList: React.FC = () => {
                   <strong>{accountAddress}</strong>
                   <small>{contractName || '--'}</small>
                 </Stack>
-
-                {isActive && <ExportButton id={account.id} typeName={typeName}/>}
```

Comment:
Do we have this ExportButton in other place?

Reply:
yes, adding in ContractList.tsx for active contract tab item, tweaking styling now

Reply:
this commit fixed this: https://github.com/onflow/flow-playground/pull/177/commits/3720f567eb86b83422308c80a1aa30903e2249dc